dali/pipeline/operators/reader/file_reader_op.cc

```diff
@@ -25,7 +25,10 @@ DALI_SCHEMA(FileReader)
   .NumInput(0)
   .NumOutput(2)  // (Images, Labels)
   .AddArg("file_root",
-      R"code(Path to a directory containing data files.)code",
+      R"code(Path to a directory containing data files.
+`FileReader` supports flat directory structure. `file_root` directory should contain
+directories with images in them. To obtain labels `FileReader` sorts directories in 
+`file_root` in alphabetical order and takes an index in this order as a class label.)code",
```

Comment:
Do you think that providing an example here would be helpful?

Reply:
There is a detailed example of this in the [`Getting started`](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/getting%20started.html#Defining-the-pipeline) notebook, so maybe no? Here I just wanted to make it clear, what is supported.